### ISSUE TITLE
Model does not display in SketchUp

### ISSUE THREAD

**Issue by macumber** [NREL/OpenStudio#1999]
Model provided by @eringold (in issues repo) does not display in SketchUp when loaded.  Double clicking spaces in the inspector causes the geometry to be shown.  My guess is something is crashing silently when opening the file so that the sketchup entities are not created

**Comment by macumber** [comment]
@eringold have you seen this on any other models?

**Comment by eringold** [comment]
@macumber no, this was a completely new occurrence to me. Note: I was able to fix it by subdividing the large, >60-surface spaces into multiple smaller spaces.

**Comment by macumber** [comment]
How did you create this geometry in the first place?  Did you import this model from gbXML or something other than creating it in SketchUp?

**Comment by eringold** [comment]
Nope, I traced over an imported dwg, then extruded with 'create spaces from diagram'.

**Comment by jmarrec** [comment]
I'm having the same issue right now. Model was displaying fine. I closed sketchup and reopened it and there's nothing displayed though I do see everything in the model inspector

**Comment by jmarrec** [comment]
If I change the version to 1.9.4 in text editor and try to open it in sketchup 2015 with OS 1.9.4 it doesn't work either.

I have many checkpoints so I pinpointed where it went wrong: right from the start.
I had a functional model in 1.9.4, that I upgraded to 1.10.1. This works.
Then my next checkpoint it already doesn't work.

Here's the changelog between the two:
- I changed the building axis to 28 deg (from -0, default): that's not the cause, I checked.
- I used the "On demand template" generator to create the midrise apartment space types and constructions (my original template is office) and checked "false" for "Set building defaults using new objects"
- I renamed about 10 spaces and assigned space types, including some for the previously generated MidRise

I deleted in text editor all the generated info from the MidRise apartment (schedules, loads, constructions, rendering colors, etc) and now I can see the geometry in sketchup...
Obviously this mean that my space types have been reset for these spaces, which is not something that I want.

Any idea?

**Comment by jmarrec** [comment]
I also tried deleting all spaces (3 plenums) with 50+ surfaces, and the model loads.

I also did the opposite: deleted all spaces except these three, and the model loads too...

I'm confused now.

**Comment by jmarrec** [comment]
The model with only the big plenums was loading.
I split each plenum in two, now it won't load anymore. I thought maybe this had to do with the fact that it opened when I saved the model last from the API in command line, so I tried that too here, didn't work.

I emailed my model to you.

**Comment by macumber** [comment]
Added second file from @jmarrec

**Comment by macumber** [comment]
Somehow the space's SU entities are being deleted:

    m = OpenStudio::Plugin.model_manager.model_interface.openstudio_model
    s= m.getSpaces[0]
    puts s.drawing_interface
      #<OpenStudio::Space:0x00000012f539c8>
    puts s.drawing_interface.entity
       #<Deleted Entity:0x12a03680>
    puts s.name
      Space 137
    s.surfaces.each {|su| puts su.drawing_interface.entity}
      #<Deleted Entity:0x1433bee0>
      #<Deleted Entity:0x144c9fa0>
      #<Deleted Entity:0x144e2988> 
      #<Deleted Entity:0x144f9750>
      #<Deleted Entity:0x148ec6a0>
      #<Deleted Entity:0x1492afe0>
      #<Deleted Entity:0x14938848>

**Comment by macumber** [comment]
Tested this occurs with OS 1.10 and SU 2016, OS 1.10 and SU 2015, OS 1.9 and SU 2014, OS 1.8 and SU 2014

**Comment by macumber** [comment]
http://forums.sketchup.com/t/groups-mysteriously-deleted-in-an-operation/19719/1

**Comment by macumber** [comment]
I spent a little more than a day looking at this and don't have any leads.  Are you able to get around this issue by simplifying geometry when tracing over DXF?

**Comment by jmarrec** [comment]
For this specific building I spent a lot of time opening an earlier version that worked, getting the info I needed (surface name for example) and performing operations using the API, like setting adiabtic surfaces etc.
After countless of them, the model started opening up in sketchup again...

I have no clue why but I was glad it did because I had windows and stuff to adjust or add, and I couldn't have done that in the dark, after 3-4 days of tinkering with it I was about to give up 

Envoyé de mon iPhone

**Comment by eringold** [comment]
I was able to work around by simplifying geometry. I appreciate your investigative effort, @macumber!

**Comment by jmarrec** [comment]
yo tambien, +1

**Comment by macumber** [comment]
I may have to close this as won't fix

**Comment by eringold** [comment]
I won't object to that. I'll just think of this as a built-in reminder to not make overly complex models!

**Comment by macumber** [comment]
Hit this again on a model that I made, re-opening
